kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandler.java

```diff
@@ -1188,7 +1176,7 @@ protected void handleSaslAuthenticate(KafkaHeaderAndRequest saslAuthenticate,
         try {
             saslAuth = SaslUtils.parseSaslAuthBytes(Utils.toArray(request.saslAuthBytes()));
 
-            namespace = NamespaceName.get(saslAuth.getUsername());
+            // saslAuth.getUsername() is useless now
```

Comment:
please add a todo here, and link the github issue?

Reply:
Ok, it's a issue of authentication, I'll add a TODO.

Reply:
also open an new github issue for the authentication not working well.

Reply:
Ok

Reply:
#208 is opened and the TODO has been added to comments